debug/debug-javascript-update.md

```diff
@@ -0,0 +1,80 @@
+# Using Your Browser to Diagnose JavaScript Errors
+
+Interactive functionality in WordPress often relies on JavaScript. If you encounter broken menus, metaboxes that won’t drag, or buttons that don’t work (such as **Add Media**), these may be caused by JavaScript errors or conflicts.
+
+This guide explains how to use your browser to diagnose JavaScript issues so you can provide better information when requesting support.
+
+---
+
+## Step 1: Try Another Browser {#try-another-browser}
+
+Different browsers sometimes handle JavaScript differently. Test your site in at least two browsers:
+
+- **If the issue only occurs in one browser** → it may be browser‑specific (e.g., an extension conflict).  
+- **If the issue occurs in all browsers** → it is likely a JavaScript or plugin/theme error.
+
+👉 Note: Browser extensions (ad blockers, privacy tools) can sometimes interfere with JavaScript. Try disabling extensions or using a private/incognito window to rule this out.
+
+Record which browsers show the error; include this information in your support request.
+
+---
+
+## Step 2: Enable SCRIPT_DEBUG {#enable-script-debug}
+
+WordPress uses minified JavaScript by default. Enabling `SCRIPT_DEBUG` forces WordPress to load the full, uncompressed versions, which makes errors easier to trace.
+
+1. Open `wp-config.php`.  
+2. Add this line **before** “That’s all, stop editing! Happy publishing”:  
+
+```php
+define( 'SCRIPT_DEBUG', true );
+```
+
+3. Reload your site and check if the error still occurs.
+
+- **If the issue is fixed** → turn off `SCRIPT_DEBUG` again and report this detail in your support request.  
+- **If the issue persists** → continue to Step 3.
+
+---
+
+## Step 3: Open Browser Developer Tools {#open-devtools}
+
+Every modern browser includes Developer Tools. Open them and switch to the **Console** tab.
+
+- **Chrome / Edge / Opera**: Press **F12** (or `Ctrl+Shift+I` / `Cmd+Option+I`) → select the **Console** tab.  
+- **Firefox**: Press **F12** (or `Ctrl+Shift+K` / `Cmd+Option+K`) → **Console** tab.  
+- **Safari**: First enable the Develop menu: `Safari → Settings → Advanced → Show Develop menu`. Then open **Develop → Show JavaScript Console**.
```

Comment:
Some versions of macOS call the option "Show features for web developers" (same location).

Also once enabled, Cmd+Option+I will open the console like other browsers.